packages/asc-ui/src/components/Accordion/AccordionWrapper.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { useRef } from 'react'
+import useFocusWithArrows from '../../utils/hooks/useFocusWithArrows'
+
+const AccordionWrapper: React.FC = ({ children }) => {
+  const ref = useRef<HTMLDivElement>(null)
+  const { keyDown } = useFocusWithArrows(ref, true)
+  return (
+    <div role="button" tabIndex={-1} onKeyDown={keyDown} ref={ref}>
```

Comment:
Does this even work? Having a negative tab index (effectively making it impossible to focus element) and defining a key down callback? Is the callback handled when the element cannot be focused?